src/render-root.ts

```diff
@@ -84,7 +84,7 @@ async function main() {
   const projectId = '${name}'
   const dataPath = config.INDEXER_DATA_PATH || undefined // 'data'
   const transport =
-    (config.INDEXER_TRANSPORT as TransportType) || TransportType.LocalNet
+    (config.INDEXER_TRANSPORT as TransportType) || TransportType.Thread
```

Comment:
```suggestion
    (config.INDEXER_TRANSPORT as TransportType) || TransportType.LocalNet
```
Should've been reset by me